### PR TITLE
Documentation/installation reorganization

### PR DESCRIPTION
**Pull request by lucieleonard**
**Component**:

Documentation

**Context**: 

Installation revamp.

**Summary**:

- Renamed quickstart into installation
- Few restructuring 
- Added the topic "Post-Installation Procedure"

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

**Comment by thomasdanan** [comment]
@lucieleonard I think you are missing png images under installation_guide
Thanks

File: docs/installation/index.rst [hunk 1]
```diff
@@ -15,5 +14,6 @@ and will show how to access various services after completing the setup.
    introduction
    setup
    bootstrap
+   post-install
```
Comment (thomasdanan) [inline docs/installation/index.rst]: This post install section should be after cluster expansion, to validate the whole cluster is well deployed and to provision volumes for prometheus and alert manager

**Comment by thomasdanan** [comment]
@lucieleonard don't you want to rename all occurrences of quickstart with installation in this PR?

**Review by thomasdanan — DISMISSED** [review]

**Review by gdemonet — APPROVED** [review]

**Pull request merged**